problemsets-mat223/linearalbegra.tex

```diff
@@ -3934,15 +3946,54 @@ \section*{Task 2.3: Pat and Jamie}
 	\end{definition}
 
 	\question
-	Let $\mathcal P$ be projection onto $\Span\{\vec u\}$ where $\vec u=\mat{2\\3}$,
+	Let $\mathcal P$ be projection onto $\Span\Set{\vec u}$ where $\vec u=\mat{2\\3}$,
 	and let $\mathcal R$ be rotation counter-clockwise by $90^\circ$.
 	\begin{parts}
 		\item Describe $\Range(\mathcal P)$ and $\Range(\mathcal R)$.
+			\begin{solution}
+				$\Range(\mathcal P)=\Span\Set{\vec u}$, and $\Range(\mathcal R)=\R^2$.
+
+				For $\mathcal P$, by definition of projections $\mathcal P(\vec x)$
+				is the vector in $\Span\Set{\vec u}$ that is closest to $\vec x$,
+				so in particular $\mathcal P(\vec x) \in \Span\Set{\vec u}$ for
+				all $\vec x\in\R^2$. Therefore $\Range(\mathcal P)\subseteq\Span\Set{\vec u}$.
+
+				On the other hand, $\mathcal P(\alpha \vec u)=\alpha\mathcal P(\vec u)=\alpha\vec u$ 
+				for any scalar $\alpha$, and so $\Range(\mathcal P)=\Span\Set{\vec u}$.
+
+				For $\mathcal Q$, we have that any vector $\vec x\in\R^2$, 
+				$\vec x=\mathcal Q(\vec y)$, where $\vec y$ is the rotation of 
+				$\vec x$ clockwise by $90^\circ$.
+			\end{solution}
 		\item What is the rank of $\mathcal P$ and the rank of $\mathcal R$?
+			\begin{solution}
+				$\Rank(\mathcal P)=1$ and $\Rank(\mathcal R)=2$.
+
+				By the previous part, we know $\Range(\mathcal P)$ is 1-dimensional
+				and $\Range(\mathcal Q)$ is 2-dimensional. 
+			\end{solution}
 		\item Let $P$ and $R$ be the matrices corresponding to $\mathcal P$ and
 			$\mathcal R$. What is the rank of $P$ and the rank of $R$?
-		\item Make a conjecture about how the rank of a transformation and
-			the rank of its corresponding matrix relate. Can you justify your claim?
+			\begin{solution}
+				$\Rank(P)=1$ and $\Rank(R)=2$.
+
+				By a previous exercise, $P=\frac{1}{13}\mat{4&6\\6&9}$ and 
+				$R=\mat{0&-1\\1&0}$ are the	matrices corresponding to $\mathcal P$
+				and $\mathcal R$. Then we compute:
+				\[
+					\rref(P)=\mat{1&\frac{3}{2}\\0&0}
+					\quad \text{and} \quad
+					\rref(R)=\mat{1&0\\0&1}.
+				\]
+				These matrices have 1 and 2 pivots, respectively. 
+			\end{solution}
+		\item Make a conjecture about how the rank of a transformation and the
+			rank of its corresponding matrix relate. Can you justify your claim?
+			\begin{solution}
+				They are equal. 
+
+				
```

Comment:
You can say that the range and column space are the same (by ...) and so their dimensions are the same. Then you can say the dimension of the column space is the same as the number of pivots in rref. (They know about pivots from the homework).

It should work to add a `\label` and `\ref` for problem numbers/parts. If it doesn't, let me know.